neurom/fst/_neuritefunc.py

```diff
@@ -81,6 +81,14 @@ def n_leaves(neurites, neurite_type=NeuriteType.all):
     return n_sections(neurites, neurite_type=neurite_type, iterator_type=Tree.ileaf)
 
 
+def total_surface_area_per_neurite(neurites, neurite_type=NeuriteType.all):
```

Comment:
to make this a feature that's available from the command line, it has to be named in
neurom/fst/__init__.py, which I think would be a good idea